app/globals.css

```
@@ -116,6 +116,54 @@
   }
 }
 
+/* Theme Toggle Animation - Shigure UI Dance Style */
+@media (prefers-reduced-motion: no-preference) {
+  ::view-transition-group(root) {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)